code/modules/mob/living/carbon/human/death.dm

```diff
@@ -101,13 +101,12 @@
 	SetJitter(0)
 	heart_attack = 0
 
+	for(var/obj/item/organ/O in organs) //these are external organs only: arms, legs, etc, internal organs are handled by carbon/death()

```

Comment:
although it could be nice as a base to something like on death exploding limbs or something. pushed with internal organs only anyway, since i don't use the external ones